bin/protractor

```diff
@@ -2,4 +2,11 @@
 
 process.env.NODE_ENV = process.env.NODE_ENV || 'test';
 
+// Transpile code to javascript prior to running javascript
+var spawn = require('child_process').spawnSync;
+var transpile = spawn('node_modules/typescript/bin/tsc');
```

Comment:
the strategy of compiling files in the client is very different from what we do elsewhere. why not ship compiled .js files to the client?